python/dgl/graphbolt/minibatch.py

```diff
@@ -500,6 +500,49 @@ def apply_to(x, device):
         return self
 
 
+    def to_pyg_adapter(self):
+        """
```

Comment:
Could you change docstring to this format? https://github.com/dmlc/dgl/blob/master/python/dgl/graphbolt/impl/fused_csc_sampling_graph.py#L515 I think it will be better for user to read.

Reply:
for sure